ivy/functional/ivy/activations.py

```diff
@@ -97,6 +97,7 @@ def deserialize(
 ]
 
 
+
```

Comment:
Any reason this extra line was added?

Reply:
I guess that this got carried forward as a glitch when I migrated the code to experimental.